application.yml

```diff
@@ -38,6 +38,10 @@ spring:
         show_sql: true
         use_sql_comments: true
         format_sql: true
+  servlet:
+    multipart:
+      location: "${java.io.tmpdir}"
+      file-size-threshold: 128KB
```

Comment:
Is this the threshold to decide to keep the processing of the uploaded file in memory or to download locally to the server file system? Is this easy to configure in a dockerized AWS hosted instance?